test/types/fastify.test-d.ts

```diff
@@ -127,7 +127,7 @@ expectAssignable<FastifyInstance>(fastify({ serverFactory: () => http.createServ
 expectAssignable<FastifyInstance>(fastify({ caseSensitive: true }))
 expectAssignable<FastifyInstance>(fastify({ requestIdHeader: 'request-id' }))
 expectAssignable<FastifyInstance>(fastify({ requestIdHeader: false }))
-expectAssignable<FastifyInstance>(fastify({ genReqId: () => 'request-id' }))
+expectAssignable<FastifyInstance>(fastify({ genReqId: (req) => req.rawHeaders[1] || 'foo' }))
```

Comment:
The actual typing test needs to be something like this:

```typescript
fastify({ genReqId: (req) => {
  expectType<RawRequestDefaultExpression>(req)
}})
```

Reply:
yeah I think they are unrelated to this change.

Reply:
Ok, I'll remove them